sklearn/metrics/_classification.py

```diff
@@ -3479,3 +3543,94 @@ def d2_log_loss_score(y_true, y_pred, *, sample_weight=None, labels=None):
     )
 
     return 1 - (numerator / denominator)
+
+
+def multiclass_brier_score_loss(y_true, y_prob, sample_weight=None, labels=None):
+    r"""Compute the Brier score loss.
```

Comment:
@ogrisel 
1. should we add a `@validate_params` here ? It seems that all metrics have this decorator now.   
2. should we rename `y_prob -> y_proba` ? It seems to be the new convention (eg in `brier_score_loss`)